packages/fiori/src/fcl-utils/FCLLayout.ts

```diff
@@ -1,22 +1,41 @@
-const getLayoutsByMedia = () => {
+enum MEDIA {
+	PHONE = "phone",
+	TABLET = "tablet",
+	DESKTOP = "desktop",
+}
+
+type LayoutConfiguration = {
+	[key in MEDIA]: {
+		[key: string]: {
```

Comment:
For me better solution here is to import FCLLayout from types. It will prevent us from adding unused layouts.
```ts

type LayoutConfiguration = {
	[key in MEDIA]: {
		[key in FCLLayout]
	};
};
```

Reply:
Since some keys arent part of FCLLayout from types, but instead are in the `getNextLayoutByStartArrow`
`getNextLayoutByEndArrow` functions, should I make them in `enum` and refactor it like this ?

```ts
type LayoutConfiguration = {
	[key in MEDIA]: {
		[key in FCLLayout & getNextLayoutByEndArrow & getNextLayoutByStartArrow]: {
			layout: Array<string>;
			arrows: Array<{
				visible: boolean;
				dir: null | string;
				separator?: boolean;
			}>;
		};
	};
};
```

Reply:
I dont think this would work, since there are some keys that are absent in FCLLayout.

Reply:
It's complaining for ThreeColumnsStartExpanded. Since layout accepts values from FCLLayout enum only and ThreeColumnsStartExpanded is not part of it,  it will fallback to the initial (default) value which is OneColumn.

I think that we can remove it.